### DISCUSSION TITLE
New release is older version than previous build

### DISCUSSION THREAD

**Discussion by sfksuperman** [arichornlover/uYouEnhanced#231] [Q&A]
Last week **v18.46.3-3.0.1** was released and today, **v18.08.1-3.0.1-F4** is released. I am confused about what to use. Why a lower version when the official YT is 18.46.3.

Please help me understand. Thanks.

**Comment by ego-lay-atman-bay** [comment]
Because some people like to use older versions, as it can remove modern stuff.

**Reply by sfksuperman** [reply]
What modern stuffs can be removed?

**Reply by ego-lay-atman-bay** [reply]
I don't really know because I prefer to use the latest version, but I'm sure someone else who knows can tell you.

**Comment by arichornlover** [comment]
> Last week **v18.46.3-3.0.1** was released and today, **v18.08.1-3.0.1-F4** is released. I am confused about what to use. Why a lower version when the official YT is 18.46.3.

I released 18.08.1 because it’s stable and a lot of people prefer this version over the latest and this release is for users that want to stop specific issues like YTSpeed, YouPiP, and other stuff that fixes it

but I don’t recommend downgrading to that release because
1. It’s not labeled as `latest`, and I suggest using the latest version based on your question
2. downloading and installing 18.08.1 can lead to YouTube corruption or crashing if you are on the latest version. Not sure how but that does happen to some.

**Reply by sfksuperman** [reply]
That's a great explanation. Thanks a lot bro, we can close this thread now.
